src/sidebars/search/search.html

```diff
@@ -1,6 +1,6 @@
 <div ng-keydown="searchCtrl.onSearchKeyDown($event)">
   <div>
-      <img src="assets/png/search_tab.png" class="top-icon" title="Search Stocks" draggable="false" />
+      <img src="assets/png/search_tab.png" ng-click="sidebarCtrl.toggleClick(); $event.stopPropagation();" class="top-icon" title="Search Stocks" draggable="false" />
```

Comment:
I'll double check this. I don't think it is actually necessary any more. I think it might just be left over from the first implementation that I tried without the `toggle` function.